luzer/CMakeLists.txt

```diff
@@ -7,8 +7,13 @@ else()
   message(FATAL_ERROR "Unsupported system: ${CMAKE_SYSTEM_NAME}")
 endif()
 
-SetClangLibPath("libclang_rt.asan-${ARCH}.a" LIBCLANG_ASAN_LIB)
-SetClangLibPath("libclang_rt.ubsan_standalone-${ARCH}.a" LIBCLANG_UBSAN_LIB)
+if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  SetClangLibPath("libclang_rt.asan_abi_osx.a" LIBCLANG_ASAN_LIB)
+  SetClangLibPath("libclang_rt.ubsan_minimal_osx.a" LIBCLANG_UBSAN_LIB)
+else()
+  SetClangLibPath("libclang_rt.asan-${ARCH}.a" LIBCLANG_ASAN_LIB)
+  SetClangLibPath("libclang_rt.ubsan_standalone-${ARCH}.a" LIBCLANG_UBSAN_LIB)
+endif()
```

Comment:
Please, leave removed lines as default variant and add a condition with a single branch for Darwin.